### DISCUSSION TITLE
Microsoft.Dynamics.Commerce.Runtime.Client

### DISCUSSION THREAD

**Discussion by spotityza** [microsoft/Dynamics365Commerce.Solutions#147] [Q&A] [answered]
Is there anything that replaces Microsoft.Dynamics.Commerce.Runtime.Client? We are using SecurityManager.Create method in an extension but cannot find any package that replaces this in Commerce SDK.

**Comment by nicwillemse** [comment]
Any solution to this?

**Comment by zhangguanghuib** [comment, marked as answer]
Is this code what you want?
    public async Task<Employee> GetEmployeePermissionsAsync(string staffId)
        {
            if (string.IsNullOrWhiteSpace(staffId))
            {
                throw new ArgumentNullException(nameof(staffId));
            }

            var request = new GetEmployeePermissionsRequest();
            request.StaffId = staffId;
            var response = await this.runtime.ExecuteAsync<GetEmployeePermissionsResponse>(request).ConfigureAwait(false);
            return response.Employee;
        }

**Reply by nicwillemse** [reply]
Worked like a charm, thanks!

**Comment by zhangguanghuib** [comment]
Please mark this question as answered
